app/src/main/res/layout/preferences_with_send_logs_layout.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="72dp"
+    android:clickable="true"
+    android:focusable="true"
+    android:foreground="?android:attr/selectableItemBackground"
+    app:cardElevation="2dp"
+    >
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:id="@+id/settings_status_background"
+      android:animateLayoutChanges="true"
+      >
+
+    <ImageView
+        android:id="@android:id/icon"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:layout_marginStart="16dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        />
+
+    <TextView
+        android:id="@android:id/title"
+        style="@style/SettingsTextStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toEndOf="@android:id/icon"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Pref Title"
+        />
+
+    <ImageView
+        android:id="@+id/settings_check_ic"
+        android:layout_width="13dp"
+        android:layout_height="13dp"
+        android:layout_marginEnd="5dp"
+        app:layout_constraintTop_toTopOf="@id/settings_status_text"
+        app:layout_constraintBottom_toBottomOf="@id/settings_status_text"
+        app:layout_constraintEnd_toStartOf="@id/settings_status_text"
+        />
+
+    <TextView
+        android:id="@+id/settings_status_text"
+        style="@style/SettingsDetailsTextStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="17dp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        tools:text="Status"
+        />
+
+    <androidx.constraintlayout.widget.Guideline
```

Comment:
Can this guideline be removed? Seems it's not doing much

Reply:
agreed, altered